Source/DataDictionary/gdcmCSAHeaderDefaultDicts.cxx

```diff
@@ -1,4 +1,4 @@
-
+cod
```

Comment:
^wotsit ?

Reply:
Sorry, I believe that's the effect of the middle button / wheel of the mouse. I guess it happens  when I scroll in the editor using the wheel, I must have pushed inadvertently, resulting in a paste.